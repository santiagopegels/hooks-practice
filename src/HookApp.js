import React from 'react'

export const HookApp = () => {
    return (
        <div>
            <h1>Hello Wooorld</h1>
        </div>
    )
}
