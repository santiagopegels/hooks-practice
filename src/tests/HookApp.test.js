import React from 'react';
import { HookApp } from '../HookApp';
import { shallow } from 'enzyme';

describe('<HookApp /> tests', () => {
    test('should show correctly', () => {

        const wrapper = shallow( <HookApp />);

        expect(wrapper).toMatchSnapshot();

    })

}) 
