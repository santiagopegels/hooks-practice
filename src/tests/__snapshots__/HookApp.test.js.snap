// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HookApp /> tests should show correctly 1`] = `
<div>
  <h1>
    Hello Wooorld
  </h1>
</div>
`;
